wp-content/themes/core/templates/page.html

```diff
@@ -1,5 +1,3 @@
 <!-- wp:template-part {"slug":"header","tagName":"header","className":"site-header"} /-->
-<!-- wp:group {"tagName":"main","layout":{"type":"constrained"}} -->
-<main class="wp-block-group"><!-- wp:post-content {"align":"full","layout":{"type":"constrained"}} /--></main>
-<!-- /wp:group -->
+<!-- wp:post-content {"align":"full","layout":{"type":"constrained"}} /-->
```

Comment:
🔴 we should keep the `main` element on the template. Is it possible to change the post-content block's tag to `main`?